src/main/java/harmonised/pmmo/config/codecs/MobModifier.java

```diff
@@ -0,0 +1,20 @@
+package harmonised.pmmo.config.codecs;
+
+import com.mojang.serialization.Codec;
+import com.mojang.serialization.codecs.RecordCodecBuilder;
+import net.minecraft.world.entity.ai.attributes.AttributeModifier;
+
+public record MobModifier(
+	String attribute,
```

Comment:
Is there a reason we don't just use `ResourceLocation` as the type here?  It' has it's own codec already we can use.

Reply:
That would require a considerable rework in MobAttributeHandler. Most of the logic there was using Strings already. I could try to change it, but that would most likely affect Nearby Players & Boss scaling as well.

Reply:
Leave it as strings then.  I'll refactor it later.